semgrep-core/src/parsing/other/yaml_to_generic.ml

```diff
@@ -56,13 +56,16 @@ type env = {
   (* From yaml.mli: "[parser] tracks the state of generating {!Event.t}
    * values" *)
   parser : S.parser;
+  anchors : (string, G.expr * E.pos) Hashtbl.t;
   mutable last_event : (E.t * E.pos) option;
 }
 
 (*****************************************************************************)
 (* Helper functions *)
 (*****************************************************************************)
 
+exception UnrecognizedAlias
```

Comment:
usually I try to add the location in the error, so of Parse_info.t,
so when we catch it somewhere we can report a location.